src/mdo/model.vm

```diff
@@ -493,40 +482,15 @@ public class ${class.name}
     #end
         @Nonnull
         public ${class.name} build() {
+            // this method should not contain any logic other than creating the object in order to ease subclassing
```

Comment:
Is that a hint that the below code should be removed ?  If so, I disagree, as this is an important optimisation.
If not, maybe the line should be removed or made more explicit...

Reply:
No it isn't, this needs to be kept as optimization. I clarified the comment in https://github.com/apache/maven/pull/1660/commits/effa73709e4b49a9509f0891ee3c6225b0b9a02c.